paper/paper.md

```diff
@@ -1,46 +1,103 @@
 ---
-title: 'splot: '
+title: 'splot: Visual Analytics for Spatial Statistical Analysis in PySAL'
 tags:
   - Python
   - visualization
   - spatial analysis
   - spatial statistics
 authors:
-  - name: XY
-    orcid: 0000-0003-0872-7098
+  - name: Stefanie Lumnitz
+    orcid: 0000-0002-7007-5812
     affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author 2
-    orcid: 0000-0000-0000-0000
+  - name: Dani Arribas-Bell
+    orcid: 0000-0002-6274-1619
+    affiliation: 3
+  - name: Renan X. Cortes
+    orcid: 0000-0002-1889-5282
+    affiliation: 2
+  - name: James D. Gaboardi
+    orcid: 0000-0002-4776-6826
+    affiliation: 4
+  - name: Verena Griess
+    orcid: 0000-0002-3856-3736
+    affiliation: 1
+  - name: Wei Kang
+    orcid: 0000-0002-1073-7781
+    affiliation: 2
+  - name: Taylor M. Oshan
+    orcid: 0000-0002-0537-2941
+    affiliation: 7
+  - name: Levi Wolf
+    orcid: 0000-0003-0274-599X
+    affiliation: "5,6"
+  - name: Sergio Rey
+    orcid: 0000-0001-5857-9762
     affiliation: 2
 affiliations:
- - name: Lyman Spitzer, Jr. Fellow, Princeton University
+ - name: University of British Columbia
    index: 1
- - name: Institution 2
+ - name: Center for Geospatial Sciences, University of California Riverside
    index: 2
-date: 13 August 2017
+ - name: Geographic Data Science Lab, Department of Geography & Planning, University of Liverpool
+   index: 3
+ - name: Pennsylvania State University
```

Comment:
I guess I should add "Department of Geography" to my affiliation?

Could you adjust to: Department of Geography, Pennsylvania State University